frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -149,6 +154,9 @@ const MutableRow: React.FC<MutableRowProps> = ({ data, onUpdate, onReturn, valid
     <TableRow key={data.id}>
       <KeyCell data={data} />
       <TableCell>
+        {data.disabledFieldlabel && (
+          <Typography variant="body4">{data.disabledFieldlabel}</Typography>
+        )}
```

Comment:
this is a hack as it should really go inside here https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/Table/metadata-table.tsx#L153-L155 but ran into styling issues. I was able to fix some but those new changes then caused other styling issues (examples below), so went with the hack for now. left a TODO to come back to fix the styling or change the design to be a text field and reset icon (like we discussed in the meeting)

**ex 1**
<img width="200" alt="Screen Shot 2022-10-25 at 9 55 29 AM" src="https://user-images.githubusercontent.com/39421794/197798755-bd4d3346-f677-4566-896a-d8f03149f8f2.png">

**ex 2**
<img width="200" alt="Screen Shot 2022-10-25 at 9 56 19 AM" src="https://user-images.githubusercontent.com/39421794/197798869-cbe51318-0a97-47ae-8908-55ffd9c1dd6e.png">

**ex 3**
<img width="200" alt="Screen Shot 2022-10-25 at 9 56 49 AM" src="https://user-images.githubusercontent.com/39421794/197798907-7e312de5-ca77-48ff-a6c4-39974ca27beb.png">